src/main/php/PDepend/Source/Language/PHP/AbstractPHPParser.php

```diff
@@ -7225,7 +7225,7 @@ function ($image) use ($useSymbolTable) {
      * @return \PDepend\Source\AST\ASTType|null
      * @since 0.9.6
      */
-    private function parseFieldDeclarationType()
+    protected function parseFieldDeclarationType()
```

Comment:
```suggestion
    private function parseFieldDeclarationType()
```
Revert useless scope change